lib/hashes.md

```diff
@@ -5,40 +5,46 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A hash is defined by curly brackets {} and can be viewed as a form of an array
+except the order or list of values are not important with hashes as it is with arrays.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+When calling for a value within a hash, a key must be used instead of an integer
+(unless an integer was used as a key).
```

Comment:
I would have to say a hash would be used if I wanted to store funny answers to small questions.
Hash_funnyanswers would be made for people who would enter Hash_funnyanswers[:life] and return 42 or Hash_funnyanswers[:love] would return "baby don't hurt me".

Other than that I am not too creative to take advantage of the "indexless" nature of hashes.